library/alloc/src/collections/btree/map.rs

```diff
@@ -167,6 +144,43 @@ pub(super) const MIN_LEN: usize = node::MIN_LEN_AFTER_SPLIT;
 /// // modify an entry before an insert with in-place mutation
 /// player_stats.entry("mana").and_modify(|mana| *mana += 200).or_insert(100);
 /// ```
+///
+/// # Background
+///
+/// A B-tree is (like) a [binary search tree], but adapted to the natural granularity that modern machines like to consume data at.
```

Comment:
Isn't this line a little too long? But I don't see tidy complaining so idk, just looks out of place 🤷

Reply:
Yeah, I think this should be wrapped.